docs/architecture/devops-for-aspnet-developers/actions-build.md

```diff
@@ -35,6 +35,18 @@ The `steps` node can be as easy as inline commands, or they can be actions. Most
 > [!TIP]
 > For more information, see [GitHub Actions YAML syntax](https://docs.github.com/actions/reference/workflow-syntax-for-github-actions).
 
+From a workflow file, you're able to `run` any of the available [.NET CLI commands](../../core/tools/index.md). For example, if you're required to build, test, and deploy an ASP.NET Core Blazor WebAssembly app with Ahead-of-Time (AoT) compilation, you'd use the following commands:
```

Comment:
I don't know if we have docs on Wasm AOT but wonder if we should link to that here?

Reply:
I'll check, if we have them - I'll add them.